cmd/puppeth/wizard_genesis.go

```diff
@@ -51,6 +51,7 @@ func (w *wizard) makeGenesis() {
 			ByzantiumBlock:      big.NewInt(0),
 			ConstantinopleBlock: big.NewInt(0),
 			PetersburgBlock:     big.NewInt(0),
+			IstanbulBlock:       big.NewInt(0),
```

Comment:
We can't enable this until Istanbul is finalized.

Reply:
Should I change it? Or take care of this PR when Istanbul is finalized?

Reply:
No, it's fine to finish this PR when Istanbul is fully defined